.jshintrc

```diff
@@ -0,0 +1 @@
+{"esversion": 9}
```

Comment:
a JSHint config is not required, we are using Typescript

Reply:
I put this in to stop VSCode from complaining should I remove it?

Reply:
What was the complaint? Sounds like a workstation issue.

Reply:
removed